src/shared/components/dropdown/BasicDropdown.scss

```diff
@@ -1,20 +1,43 @@
 .basic-dropdown {
-  min-width: 100%;
-  button {
-    min-width: 100%;
-  }
+  width: 100%;
+  position:  relative !important;
+}
+
+.dropdown-toggle {
+  display: flex;
+  align-items: center;
+  padding: 8px 16px;
+  border: 1px solid #d2d2d2;
+  border-radius: 4px;
+  background-color: white;
+  cursor: pointer;
+  min-height: 36px;
+}
```

Comment:
We shouldn't be using fixed values for custom styling, Please use PF variables. Also you shouldn't need to override pf, default styling from patternfly should be enough.

Reply:
Yeah, you're right. I have replaced formats with the pf style. Could you help to review again?
BTW, it seems I have to add more styles than previous scss.  If there is anything to make it shorter, please help to advice. Thank you so much.